### PR TITLE
Day 15: unit tests for Sevices and EventListeners

### PR DESCRIPTION
**Pull request by sergey-bleih**
Maybe it makes sense to merge them little by little, to avoid large PR's. These cover `App\Service` and `App\EventListeners`. Please let me know what you think. I also added vfs for testing fs manipulations. Take a look if you think it fits.

### CODE REVIEW

File: tests/EventListener/AffiliateTokenListenerTest.php [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\EventListener;
```
Comment (gregurco) [inline tests/EventListener/AffiliateTokenListenerTest.php]: Is it ok to keep tests in the same namespace as whole application? In Symfony docs I see `App\Tests\...`.

Comment (sergey-bleih) [reply tests/EventListener/AffiliateTokenListenerTest.php]: Fixed the namespace

File: composer.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
         "friendsofsymfony/user-bundle": "~2.1",
         "jms/serializer-bundle": "^3.1",
         "knplabs/knp-paginator-bundle": "^3.0",
+        "mikey179/vfsStream": "^1.6",
```
Comment (gregurco) [inline composer.json]: Is this package required in non-dev section? I see it's used only in tests.

File: docker-compose.yml [hunk 3]
```diff
@@ -22,7 +22,7 @@ services:
             - .:/application
             - ./docker/nginx/nginx.conf:/etc/nginx/conf.d/default.conf
         ports:
-            - 80:80
+            - 82:80
```
Comment (gregurco) [inline docker-compose.yml]: no need to commit this change

**Comment by gregurco** [comment]
@sergey-bleih thank you! Great job :+1:

**Pull request merged**